### PR TITLE
Clean dependencies and improve installation part of the  README

### LINKED ISSUE #5: missing dependencies
Moment and other dependencies are declared as peerdepency but nor in normal dependencies: 

```
Module not found: Error: Cannot resolve module 'moment' in .../node_modules/@gandi/react-translate/dist/tools
resolve module moment in .../node_modules/@gandi/react-translate/dist/tools
...
@ ./~/@gandi/react-translate/dist/tools/createTranslator.js 15:14-31
```

### PR DESCRIPTION
**Pull request by iamdey**
fix #5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 ### Added
 
-  - Create a launcher for extract_messages, update_catalog and other scripts
+  - Create a launcher for extract_messages, update_catalog and other scripts.
+
+### Changed
+
+  - Clean dependencies and improve installation part of the README.
```
Comment (pascalduez) [inline CHANGELOG.md:11]: Stars for bullet points

**Review by pascalduez — APPROVED** [review]

**Review by yanndinendal — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (iamdey) [reply CHANGELOG.md:11]: it is not consistent and there is a majority of `-`. I'll not change this here. 

- dashes are well evaluated as bullets
- dashes are used in keepachangelog examples
- but yeah I usually use stars

Comment (pascalduez) [reply CHANGELOG.md:11]: Humm, you're right, keepachangelog uses dashes. Why on earth did we diverged... ? Anyway.

**Comment by ziir** [comment]
Oh, we kept lodash

**Comment by pascalduez** [comment]
> Oh, we kept lodash

It's used quite extensively on this lib, not a small refactor I would say...

**Pull request merged**